src/OrleansTestingHost/StorageTestConstants.cs

```diff
@@ -78,6 +78,19 @@ static StorageTestConstants()
             DataConnectionString = DEFAULT_STORAGE_DATA_CONNECTION_STRING;
         }
 
+        public static bool UsingAzureLocalStorageEmulator
+        {
+            get
+            {
+                string connString = DataConnectionString;
+                bool usingLocalWAS = DEFAULT_STORAGE_DATA_CONNECTION_STRING.Equals(connString, StringComparison.OrdinalIgnoreCase);
+                string msg = string.Format("Using Azure local storage emulator = {0}", usingLocalWAS);
+                Console.WriteLine(msg);
+                Trace.WriteLine(msg);
+                return usingLocalWAS;
```

Comment:
I think it was something to do with diagnosing problems with the Azure storage configs on the old TFS build servers, but not sure how important it is to have this now.